### DISCUSSION TITLE
2008/08/11/%E5%9C%A8%E5%AD%97%E7%AC%A6%E4%B8%B2%E4%B8%AD%E6%98%BE%E7%A4%BA%E5%A4%9A%E7%A7%8D%E5%AD%97%E4%BD%93

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#236] [Announcements]
# 2008/08/11/%E5%9C%A8%E5%AD%97%E7%AC%A6%E4%B8%B2%E4%B8%AD%E6%98%BE%E7%A4%BA%E5%A4%9A%E7%A7%8D%E5%AD%97%E4%BD%93

程序要求：输出一个字符串“LabVIEW真好用！”，要突出强调“好用”两个字。

https://blog.qizhen.xyz/2008/08/11/%E5%9C%A8%E5%AD%97%E7%AC%A6%E4%B8%B2%E4%B8%AD%E6%98%BE%E7%A4%BA%E5%A4%9A%E7%A7%8D%E5%AD%97%E4%BD%93

**Comment by ruanqizhen** [comment]
之前博客下的留言：

辉
2008年10月12日 14:50 编辑
我的妈，非主流也玩LABVIEW
 
之前用这个属性节点做了个实验，依次增大后面每个字的字号大小，
感觉是比较耗时

回复
ωｪййｪё
2008年08月11日 20:09 编辑
一會兒我也去試試看
看上去不錯的說呢
多謝推薦

回复
